clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -996,7 +996,7 @@ public void subscribe(Pattern pattern, ConsumerRebalanceListener listener) {
         acquireAndEnsureOpen();
         try {
             throwIfNoAssignorsConfigured();
-            log.debug("Subscribed to pattern: {}", pattern);
+            log.info("Subscribed to pattern: '{}'", pattern);
```

Comment:
Is there a reason why you added quotes here but not elsewhere?

Reply:
The regex may have funny characters. I thought it would be helpful to have it clearly delineated.